cerberus/client.py

```diff
@@ -53,8 +54,8 @@ def _add_slash(self, string=None):
             return str.join('', [string, '/'])
         return str(string)
 
-    def set_lambda_context(self, lambda_context):
```

Comment:
Is this required as part of the token override, or is this unrelated?

Reply:
This is required as part of the token override because it used to look like
`if lambda_context is not None:
            self.set_lambda_context(lambda_context)
        self.set_token()
`
iI's easier to skip after it gets moved into set_token().